.vscode/launch.json

```diff
@@ -21,7 +21,8 @@
 				//"--noLazy",
 				"--extensionDevelopmentPath=${workspaceFolder}/inlang/source-code/ide-extension",
 				// change this path to specify what folder should be opened upon running this configuration
-				"${workspaceFolder}/inlang/source-code/end-to-end-tests/inlang-nextjs"
+				// "${workspaceFolder}/inlang/source-code/end-to-end-tests/inlang-nextjs"
+				"/Users/jldec/inlang/paraglide"
```

Comment:
@jldec seems like you committed machine-specific (your machine) config to the entire repo?

Reply:
> @jldec seems like you committed machine-specific (your machine) config to the entire repo?

Yes, sorry @samuelstroschein (my brain likes to type `git commit -a -m`) 
I noticed  and reverted it in the merged branch.
thanks for the heads-up tho.